osu.Game/Overlays/Mods/ModSelectOverlay.cs

```diff
@@ -506,41 +493,32 @@ private void updateCustomisation()
 
             if (anyCustomisableModActive)
             {
-                customisationVisible.Disabled = false;
+                customisationPanel.Show();
 
-                if (anyModPendingConfiguration && !customisationVisible.Value)
-                    customisationVisible.Value = true;
+                if (anyModPendingConfiguration)
+                    customisationPanel.Expanded.Value = true;
             }
             else
             {
-                if (customisationVisible.Value)
-                    customisationVisible.Value = false;
-
-                customisationVisible.Disabled = true;
+                customisationPanel.Expanded.Value = false;
+                customisationPanel.Hide();
```

Comment:
Wondering if this should just always show but in a disabled state (like old behavior with the footer button).

Feels weird to me to have it hidden as a first impression (like the feature disappeared / isn't there). And when enabling Difficulty Adjust for the first time, I don't have a sense of where the panel will appear/expand.

Can also clarify further with a disabled tooltip saying `Select at least one mod that is customisable.` or something.

Reply:
Applied in https://github.com/ppy/osu/pull/28553/commits/1d94c96a8efef24d5ae54aa7159f6e7a615a50a8